tensorflow/c/c_test_util.h

```diff
@@ -33,16 +35,18 @@ typedef std::unique_ptr<TF_Tensor, decltype(&TF_DeleteTensor)>
 // Create a tensor with values of type TF_INT8 provided by `values`.
 TF_Tensor* Int8Tensor(const int64_t* dims, int num_dims, const char* values);
 
-TF_Tensor* Int32Tensor(int32_t v);
+TF_Tensor* Int32Tensor(int32_t v, const tensorflow::TensorShape& shape = {});
 
 TF_Operation* Placeholder(TF_Graph* graph, TF_Status* s,
-                          const char* name = "feed");
+                          const char* name = "feed",
+                          const tensorflow::PartialTensorShape& shape = tensorflow::PartialTensorShape());
```

Comment:
Long line?

Reply:
What is your convention for breaking these lines? In my code I usually break after the opening parenthesis of the argument list.

Reply:
Here's what clang-format gives me:
```
TF_Operation* Placeholder(TF_Graph* graph, TF_Status* s,
                          const char* name = "feed",
                          const tensorflow::PartialTensorShape& shape =
                              tensorflow::PartialTensorShape());
```

FYI I highly recommend using clang-format (set to Google style). There's an emacs plugin I use, and maybe there are plugins for other editors too. I can provide more info if necessary.

Reply:
Thanks! I'll look into that. I'm used to IntelliJ for coding in Java, Scala, or Python and I don't really have a preferred editor for C++ as I don't use it as much. I've started using CLion recently, which I like, but I feel it's not very mature yet.

Reply:
It looks like you might able to use it from clion: https://stackoverflow.com/questions/34648255/using-clang-format-in-clion. I think this requires installing the standalone version of clang-format, which you can possibly use directly to at least see its formatting suggestions and manually apply them. I'm not sure though. Anyway, maybe worth looking into depending on how hard manually formatting is.